analyzerScripts/install.ps1

```diff
@@ -0,0 +1,58 @@
+param($installPath, $toolsPath, $package, $project)
```

Comment:
To tell you the truth, I haven't looked too closely at them. My PS isn't that great either though, so I'd prefer to keep what VS auto-generated since I can just blame them if there are bugs :)

Reply:
Didn't realize they were auto gen.  Leaving them as is is okay :)